.github/workflows/publish.yml

```diff
@@ -53,3 +53,33 @@ jobs:
           push: true
           tags: ${{ steps.docker_metadata.outputs.tags }}
           labels: ${{ steps.docker_metadata.outputs.labels }}
+
+  dockerhub-description:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+
+      - name: Find change in docker readme file
+        id: docker-readme-change
+        uses: tj-actions/changed-files@v24
+        with:
+          files: |
+            docker-readme.md
```

Comment:
Hmm we do not have it yet 🤔

Reply:
In the description :wink: [PR](https://github.com/okp4/okp4d/pull/130) : https://github.com/okp4/okp4d/pull/130/commits/137e3c5f39e693bd000a1c49ce9f05ebb0508ad1